### DISCUSSION TITLE
Crop_region bug in NSI implementation

### DISCUSSION THREAD

**Discussion by aFigueroaBengoa** [cabouman/mbirjax#48] [Q&A]
Hello Professor Bouman and team, 

I am testing out some of your NSI implementation on our datasets as we also use NSI equipment at the lab. A bug that I ran into is when designating a crop region in the preprocess tools "preprocess.nsi.load_scans_and_params", the defect_list from NSI is not cropped along with the data and causes the sinogram generation to crash as it will try to correct for points that are outside the range of the cropped images.

**Comment by gbuzzard** [comment]
Can you include the code that you're using to do the cropping?  Ideally, a reproducible example, perhaps using our demo data, would be great.
